procrastinate/jobs.py

```diff
@@ -87,6 +91,9 @@ def from_row(cls, row: Dict[str, Any]) -> "Job":
             attempts=row["attempts"],
         )
 
+    def asdict(self) -> types.JSONDict:
```

Comment:
it's not a JSONDict, one of the values is a datetime. It's a `Dict`, and I'm not sure it's worth being more precise. `Dict[str, Any]` if you really want.

Reply:
(fixed it)